classes/patterns/class-issues.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/**
+ * Issues class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * Issues pattern includes:
+ * Column, Image, Heading, Paragraph, Media & Text.
+ *
+ * @package P4GBKS\Patterns
+ */
+class Issues extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/issues';
+	}
+
+	/**
+	 * Returns the pattern config.
+	 */
+	public static function get_config(): array {
+		return [
+			'title'      => __( 'Issues', 'planet4-blocks-backend' ),
+			'keywords'   => [ 'column', 'image', 'heading', 'paragraph', 'media', 'text' ],
```

Comment:
Those keywords need to be translated 🤔 Also, who decided on which ones to add?

Reply:
> who decided on which ones to add?
Nobody, I've added them since these are the used blocks 😶

Reply:
Or we could ask the design team if they have any UX/UI recommendations for this